src/igvfd/tests/test_types_tissue.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+
+def test_summary(testapp, tissue, human_donor, rodent_donor, parent_rodent_donor_2, sample_term_brown_adipose_tissue, treatment_chemical):
+    res = testapp.get(tissue['@id'])
+    assert res.json.get('summary') == 'adrenal gland tissue, male Mus musculus strain1'
+    testapp.patch_json(
+        tissue['@id'],
+        {
+            'lower_bound_age': 10,
+            'upper_bound_age': 10,
+            'age_units': 'month'
+        }
+    )
+    res = testapp.get(tissue['@id'])
+    assert res.json.get('summary') == 'adrenal gland tissue, male Mus musculus strain1 (10 months)'
+    testapp.patch_json(
+        human_donor['@id'],
+        {
+            'sex': 'male',
+        }
+    )
+    testapp.patch_json(
+        tissue['@id'],
+        {
+            'lower_bound_age': 50,
+            'upper_bound_age': 100,
+            'age_units': 'day',
+            'virtual': True,
+            'embryonic': True,
+            'donors': [human_donor['@id']],
+            'sample_terms': [sample_term_brown_adipose_tissue['@id']],
+        }
+    )
+    res = testapp.get(tissue['@id'])
+    assert res.json.get('summary') == 'virtual embryonic brown adipose tissue, male Homo sapiens (50-100 days)'
+    testapp.patch_json(
+        rodent_donor['@id'],
+        {
+            'sex': 'female',
+        }
+    )
+    testapp.patch_json(
+        parent_rodent_donor_2['@id'],
+        {
+            'sex': 'male',
+        }
+    )
+    treatment_summary = testapp.get(treatment_chemical['@id']).json.get('title')
+    treatment_summary = treatment_summary.replace('Treatment of', 'treated with')
+    treatment_summary
+    testapp.patch_json(
+        tissue['@id'],
+        {
+            'donors': [rodent_donor['@id'], parent_rodent_donor_2['@id']],
+            'treatments': [treatment_chemical['@id']]
+        }
+    )
+    res = testapp.get(tissue['@id'])
+    assert res.json.get(
+        'summary') == f'virtual embryonic brown adipose tissue, mixed sex Mus musculus strain1, strain3 (50-100 days) {treatment_summary}'
```

Comment:
i think the assertion should include the treatment summary written down, not parametrized. if I understand correctly the treatment summary will be capitalized and will appear "disconnected" from the tissue summary. but I am not sure that is true.

Reply:
I'll change this.

Reply:
ok